modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/SyncmlParser.java

```diff
@@ -321,17 +321,17 @@ private static Status generateStatus(Node node) {
                     status.setCommand(command);
                     break;
                 case PluginConstants.SyncML.SYNCML_CHAL:
```

Comment:
1.renamed SYNCML_CHAL to SYNCML_CHALLENGE
2. initiate variable to get node.getChildNodes().item(x)
from https://github.com/wso2/product-mdm/pull/156